autometrics/src/objectives.rs

```diff
@@ -119,6 +122,10 @@ impl Objective {
 }
 
 /// The percentage of requests that must meet the given criteria (success rate or latency).
+#[cfg_attr(
+    feature = "prometheus-client",
+    derive(Clone, Debug, PartialEq, Eq, Hash)
```

Comment:
Hmm, true. Do you think we should derive all of those traits all of the time (not just when using `prometheus-client`)? I'm somewhat hesitant to do that proactively because it technically expands the surface area of the API and (maybe?) increases compile times. But maybe it's fine to just do it for this type of enum

Reply:
saving `Copy` makes it way nicer to use. just be sure that u dont plan to add more in the future which may not be `Copy`, as removing `Copy` is a breaking change and requires a major version bump (unlike adding it)